packages/pds/src/oauth/oauth-scopes.ts

```diff
@@ -0,0 +1,8 @@
+export function isValidPermission(value) {
```

Comment:
I think this may be an implicit any.
```suggestion
export function isValidPermission(value: unknown) {
```